test/user_authorization_test.go

```diff
@@ -139,4 +139,58 @@ func TestUserAuthorizationProto(t *testing.T) {
 	expectResult(t, c, permErrRe)
 	sendProto(t, c, "SUB foo.baz 1\r\n")
 	expectResult(t, c, permErrRe)
+
```

Comment:
You may want to have a test with more than 1 deny and create several subscriptions that are impacted individually by those deny clauses.

Reply:
ok will look at expanding.

Reply:
Well, comment was based on my misunderstanding, but having more than one deny clause and subs may not be a bad thing.

Reply:
Agree.